src/main/java/org/usfirst/frc4904/robot/commands/controlpanel/StopPanelMotor.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc4904.robot.commands.controlpanel;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.robot.RobotMap;
+
+/**
+ * A command that will stop the control panel motor.
+ */
+public class StopPanelMotor extends CommandBase {
```

Comment:
should this be a `MotorIdle` or `MotorConstant`? Does it even need to exist? Shouldn't the motor default to `MotorIdle` anyway? Can you just cancel the command running on it?